pyanaconda/progress.py

```diff
@@ -40,6 +40,7 @@
 progressQ.addMessage("complete", 0)
 progressQ.addMessage("quit", 1)             # exit_code
 
+# TODO: this function is never used
```

Comment:
I think we won't use this code much in a future. Remove

Reply:
Removed.